MH4U Dex/CoreDataController.m

```diff
@@ -93,33 +131,33 @@ - (void)loadMonsterData
     // Because monsters are added first, there are no relationships to set up, so we can hydrate them.
     NSString *path = [[NSBundle mainBundle] pathForResource:MonstersFileName ofType:JSON];
     NSDictionary *attributes = @{
-                                 @"id":@"_id",
-                                 @"monster_class":@"class",
-                                 @"name":@"name",
-                                 @"jpn_name":@"jpn_name",
-                                 @"sort_name":@"sort_name",
-                                 @"trait":@"trait",
-                                 @"icon":@"icon_name",
-                                 @"url":@"url"
+                                 @"id": @"_id",
+                                 @"monster_class": @"class",
+                                 @"name": @"name",
+                                 @"jpn_name": @"jpn_name",
+                                 @"sort_name": @"sort_name",
+                                 @"trait": @"trait",
+                                 @"icon": @"icon_name",
+                                 @"url": @"url",
                                  };
     [self.managedObjectContext hydrateStoreWithJSONAtPath:path attributeMappings:attributes forEntityName:[Monster entityName]];
     path = [[NSBundle mainBundle] pathForResource:MonsterDamageZonesFileName ofType:JSON];
     attributes = @{
-                   @"id":@"_id",
-                   @"monster_id":@"monster_id",
-                   @"monsterName":@"monster_name",
-                   @"jpnMonsterName":@"jpn_monster_name",
-                   @"bodyPart":@"body_part",
-                   @"cut":@"cut",
-                   @"impact":@"impact",
-                   @"shot":@"shot",
-                   @"fire":@"fire",
-                   @"water":@"water",
-                   @"thunder":@"thunder",
-                   @"ice":@"ice",
-                   @"dragon":@"dragon",
-                   @"ko":@"ko",
-                   @"extract":@"extract"
+                   @"id": @"_id",
+                   @"monster_id": @"monster_id",
+                   @"monsterName": @"monster_name",
+                   @"jpnMonsterName": @"jpn_monster_name",
+                   @"bodyPart": @"body_part",
+                   @"cut": @"cut",
+                   @"impact": @"impact",
+                   @"shot": @"shot",
+                   @"fire": @"fire",
+                   @"water": @"water",
+                   @"thunder": @"thunder",
+                   @"ice": @"ice",
+                   @"dragon": @"dragon",
+                   @"ko": @"ko",
+                   @"extract": @"extract",
```

Comment:
Do constants exist for any of these strings?  Perhaps mogenerator-generated attribute names?